README.md

```diff
@@ -186,67 +186,67 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** Describe what is happening in the code above.
 
-* **Your Answer:**
+* **Your Answer:** If the state of the app has a current user then the UsersContainer component will render else the router will redirect to the login page.
 
 ---
 
 - [ ] Now try logging in. Then, when you're on the `/users` page, refresh the page.
 
 * **Question:** What happens and why?
 
-* **Your Answer:**
+* **Your Answer:** I can't quite remember what was happening before we changed the code.
```

Comment:
Previously this route, did not require us to be logged in to view. Now it will only show if we are logged in but we need to delay the rendering until we finish authentication to avoid errors.